arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,837 @@
+        },
+        "state": {
+          "type": "string",
+          "description": "Gets or sets the status of account.",
+          "enum": [
+            "Ok",
+            "Unavailable",
+            "Suspended"
+          ],
+          "x-ms-enum": {
+            "name": "AutomationAccountState",
+            "modelAsString": true
+          }
+        },
+        "creationTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the creation time."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the last modified time."
+        },
+        "description": {
+          "type": "string",
+          "description": "Gets or sets the description."
+        }
+      },
+      "description": "Definition of the account property."
+    },
+    "AutomationAccount": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AutomationAccountProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets the automation account properties."
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets or sets the etag of the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Definition of the automation account type."
+    },
+    "AutomationAccountCreateOrUpdateProperties": {
+      "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "Gets or sets account SKU."
+        }
+      },
+      "description": "The parameters supplied to the create or update account properties."
+    },
+    "AutomationAccountCreateOrUpdateParameters": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AutomationAccountCreateOrUpdateProperties",
+          "x-ms-client-flatten": true,
+          "description": "Gets or sets account create or update properties."
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the name of the resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets the tags attached to the resource."
+        }
+      },
+      "description": "The parameters supplied to the create or update automation account operation."
+    },
+    "AutomationAccountListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AutomationAccount"
+          },
+          "description": "Gets or sets list of accounts."
```

Comment:
Leave a coHi Vishrut,
 We have received exceptions for the below issues:
 [Azure Automation] validations (size, range, regexes) on properties and parameters missing  #1020
#1020
 [Azure Automation] Descriptions need to be corrected in all Azure Automation resources  #1021
#1021

Samer is aware of above issues. The two issues above will be revisited. These descriptions were inherited from Hyak specs and we won't be changing descriptions or validation regex at the moment
mment